tools/postgres/prepareTestDB.sh

```diff
@@ -12,7 +12,7 @@ dbHost="$("$scriptdir"/db_host.sh)"
 for file in $(find "$scriptdir"/../../test/db_postgres -name "*.csv")
 do
   echo $file
-  PGPASSWORD=postgres psql -U postgres -h $dbHost --dbname=$dbName -c "COPY webknossos.$(basename $file .csv) FROM STDOUT WITH CSV HEADER QUOTE ''''" < $file
+  PGPASSWORD=postgres psql -U postgres -h $dbHost --dbname=$dbName -c "SET session_replication_role = replica;COPY webknossos.$(basename $file .csv) FROM STDOUT WITH CSV HEADER QUOTE ''''" < $file
```

Comment:
`SET session_replication_role = replica;` what is thsi?

Reply:
temporarily disables triggers

Reply:
👍